src/rawchannel.cc

```diff
@@ -64,7 +64,8 @@ using namespace v8;
 #define rtr_symbol      SYMBOL("rtr")
 #define err_symbol      SYMBOL("err")
 #define data_symbol     SYMBOL("data")
-#define canfd_symbol    SYMBOL("canfd")        
+#define canfd_symbol    SYMBOL("canfd")
+#define fdflags_symbol  SYMBOL("fd_flags")
```

Comment:
Please reconsider the approach. The basic idea of this API is to hide any kernel headers, so instead of pass a bit field of fdflags, I'd prefer to provide the individual flags. E.g. if someone wanna use bitrate switch you might call send(brs: True) instead of send(fdflags:3).